api/notd/notd_message_processor.py

```diff
@@ -69,6 +71,17 @@ async def process_message(self, message: SqsMessage) -> None:
             messageContent = UpdateActivityForCollectionMessageContent.parse_obj(message.content)
             await self.notdManager.update_activity_for_collection(address=messageContent.address, startDate=messageContent.startDate)
             return
+        if message.command == UpdateTokenAttributesForAllCollectionsMessageContent.get_command():
+            if message.postDate is None or message.postDate < date_util.datetime_from_now(seconds=-(60 * 10)):
```

Comment:
this is gonna run every hour, please update this accordingly

Reply:
could you do the same for `UpdateListingsForAllCollections` below as well please